### PR TITLE
MODINVSTOR-451 Remove old data after the whole Preceding/Succeeding titles migration

### PR DESCRIPTION
**Pull request by Svitlana-Zmiivska**
https://issues.folio.org/browse/MODINVSTOR-451

**Purpose:** 

Remove old data after the whole Preceding/Succeeding titles migration:

**Approach:**

Remove preceding/succeeding data from:

- reference and sample data
- instance_relationship and instance_relationship_type tables

### CODE REVIEW

File: sample-data/instance-relationships/american-bar-association-preceding-title.json [hunk 1]
```diff
@@ -1,6 +0,0 @@
-{
```
Comment (bohdan-suprun) [inline sample-data/instance-relationships/american-bar-association-preceding-title.json]: Could you please create the same relation but for the new model?

File: src/main/resources/templates/db_scripts/removeOldPrecedingSucceedingTitles.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+START TRANSACTION;
+
+DELETE FROM ${myuniversity}_${mymodule}.instance_relationship
+WHERE instanceRelationshipTypeId ='cde80cc2-0c8b-4672-82d4-721e51dcb990';
```
Comment (marcjohnson-kint) [inline src/main/resources/templates/db_scripts/removeOldPrecedingSucceedingTitles.sql]: Are any existing records migrated to the new structure before they are deleted?

Comment (Svitlana-Zmiivska) [reply src/main/resources/templates/db_scripts/removeOldPrecedingSucceedingTitles.sql]: The migration has already been done https://github.com/folio-org/mod-inventory-storage/pull/404

Comment (marcjohnson-kint) [reply src/main/resources/templates/db_scripts/removeOldPrecedingSucceedingTitles.sql]: Ok, thank you

**Pull request merged**